README.md

```diff
@@ -1,6 +1,42 @@
 # About Power Flows DMN Model
 Power Flows DMN Model
 
+# DMN model
+Power Flows model has been designed as a YAML file. The file contains information about input and output data. The additional division is sections with fields and decisions.
+
+```yaml
+id: sample_decision_id
+name: Sample Decision Name
+hit-policy: UNIQUE
+fields:
+  in:
+  - name: age
+    type: integer
+    description: This is something about age
+  - name: colour
+    type: string
+    description: This is something about colour
+  out:
+  - name: allow
+    type: boolean
+    description: We expect a decision, if we have access to do it
+decisions:
+  in:
+  - age: 3
+  out:
+  - allow: true
+  in:
+  - age: 8
+    colour: red
+  out:
+  - allow: true
+  in:
+  - colour: green
+  out:
+  - allow: true
+    description: Green allows always
+```
```

Comment:
```yaml
id: sample_decision_id
name: Sample Decision Name
hit-policy: UNIQUE
fields:
  in:
    age:
      type: integer
      description: This is something about age
    color:
      type: string
      description: This is something about colour
  out:
    allow:
      type: boolean
      description: We expect a decision, if we have access to do it
decisions:
  - summary: Decision one
    in:
      age: 3
      color: red
    out:
      allow: true
  - summary: Another decsion
    in:
      age:
        expression: gt 3
        expression-type: SpEL
    out:
      allow: false
```

Reply:
More readable yaml.
Remember that in value can be evaluated by expression language and decision value can be result of expression too

Reply:
I think, we need also connections between DMNs, i.e. use evaluation result of one as an input to another one.

Reply:
Instead of of summary let's use annotation and @stlhrt what in a case when there is no summary? It looks like required in your proposal.

Reply:
it is not, its just a field in object being element of the list, I put it first as its a nice description of rule

Reply:
as for decision chaining: yes, we need it but it a food for thought right now